superset-frontend/src/components/ButtonGroup/ButtonGroup.stories.tsx

```diff
@@ -49,11 +49,13 @@ InteractiveButtonGroup.args = {
 InteractiveButtonGroup.argTypes = {
   buttonStyle: {
     name: STYLES.label,
-    control: { type: 'select', options: STYLES.options },
+    control: { type: 'select' },
+    options: STYLES.options,
   },
   buttonSize: {
     name: SIZES.label,
-    control: { type: 'select', options: SIZES.options },
+    control: { type: 'select' },
+    options: SIZES.options,
```

Comment:
This story works great too! I'll stop making noise now... no news is good news for any story from here on out :D